nailgun-server/src/main/java/com/martiansoftware/nailgun/NGSession.java

```diff
@@ -172,6 +172,8 @@ private Socket nextSocket() {
             // read() calls throw SocketTimeoutException which unblocks read(). The exception must
             // be properly handled by calling code.
             try {
+                // TODO(buck_team): this does not work with current NGUnixDomainSocket
```

Comment:
nit: it should probably be a todo for a nailgun team instead of buckteam :)

Reply:
Nailgun title page says that NailGun is currently supported by buck team, thus todo :)